src/app/common/data-card/DataCard.vue

```diff
@@ -2,18 +2,16 @@
   <KCard
     class="data-card"
   >
-    <template #body>
-      <dl>
-        <div class="card">
-          <dt class="title">
-            <slot name="title" />
-          </dt>
-          <dd class="body">
-            <slot name="default" />
-          </dd>
-        </div>
-      </dl>
-    </template>
+    <dl>
+      <div class="card">
+        <dt class="title">
+          <slot name="title" />
+        </dt>
+        <dd class="body">
+          <slot name="default" />
+        </dd>
+      </div>
+    </dl>
```

Comment:
CSS needs adding to set these font-sizes back to the smaller `-20` (I think its `-20`)  now that kcard doesn't do that

Reply:
Shouldn’t those cards use the same font size as KCard?

Reply:
Not according to the designs I don't think, the size for the data here is smaller than normal body copy

Reply:
I placed an override in the component. Worth checking with design whether the changes to KCard should be reflected in the design for these components or not.